solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -8,7 +10,11 @@
      * @param externalQuotesService Сервис-источник актуальных котировок
      * @return объект - валютный калькулятор
      */
-    public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    public FxConversionService getFxConversionService(@NonNull ExternalQuotesService externalQuotesService) {
+        return new FxConversionServiceImpl(externalQuotesService);
+    }
+
+    public ExtendedFxConversionService getExtendedFxConversionService(@NonNull ExternalQuotesService externalQuotesService) {
```

Comment:
Line is longer than 120 characters (found 125).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1039332978>*